Resources/Locale/en-US/prototypes/catalog/fills/crates/cargo-crates.ftl

```diff
@@ -0,0 +1,2 @@
+ent-CrateCargoQuartermasterHardsuit = luxury mining hardsuit crate
+    .desc = Finally, a hardsuit Quartermasters could call their own. Centcomm has heard you, now stop asking.
```

Comment:
Remove the reference to this being for the QM.

Reply:
Same as hardsuits.yml comment - removing the spine of the PR

Reply:
This is similarly fine.